kusd/validator/validator.go

```diff
@@ -71,7 +73,7 @@ type validator struct {
 
 	walletAccount accounts.WalletAccount
 
-	network ValidationNetwork
+	election network.Election // consensus election
```

Comment:
I find it confusing to have `validator.election` using `network.Election` and also `Election` from the current package a few lines above. Not sure what is what. Could we use a more specific naming to differentiate them? Maybe for a future refactor PR.

Reply:
still don't understand whats what :)

maybe we should not call Election the network one, and just network.contract,  for me moment or until better name comes

the embedded Election it's just data, no behaviour :|, also need cleanup most is not in use. `VotingState`?

Reply:
yes, the contract one is the real election and the other one as you say is election internal state.
I'd probably change the internal state name - VotingState sounds good.